exporter/exporterhelper/queue_sender_test.go

```diff
@@ -325,17 +328,14 @@ func TestQueuedRetryPersistentEnabled_shutdown_dataIsRequeued(t *testing.T) {
 
 	// Invoke queuedRetrySender so the producer will put the item for consumer to poll
 	require.NoError(t, be.send(newErrorRequest(context.Background())))
-
 	// first wait for the item to be produced to the queue initially
-	assert.Eventually(t, func() bool {
+	require.Eventuallyf(t, func() bool {
```

Comment:
Can I ask you to use `EventuallyWithT` instead? 
Makes it easier to debug issues instead of getting `"condition was never met"`

Reply:
sure